hearth/spec/hearth/http/error_parser_spec.rb

```diff
@@ -23,12 +23,20 @@ class ApiClientError < ApiError; end
       class ApiServerError < ApiError; end
 
       class ModeledError < ApiClientError; end
+
+      module Parsers
```

Comment:
I think this should be under TestParsers, next to TestErrors? (Errors and Parsers are siblings right?)

Reply:
Was trying to avoid extra namespaces.  The module arrangement doesn't really matter here, though I guess technically it doesn't reflect what we generate.  To be fully accurate we would need:

```
module TestService
  module Errors
     class ModeledError < ApiClientError; end # and all the other ones
   end

  module Parsers
      class ModeledError; end
  end
end
```

But it didn't really seem worth it to create more modules/nest more deeply here.

Reply:
In general I didn't like that we pollute the test namespace. There must be a fix.. maybe we should make a unit test service somehow? I guess like an API helper.